dist/vendor.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+CARGO_CONFIG=".cargo/config.toml"
+FILES_TO_COMMIT=(
+  "${CARGO_CONFIG}"
+  "vendor"
+  "rh-manifest.txt"
+)
+
+# Don't overwrite any existing config
+[ ! -f "${CARGO_CONFIG}" ] || {
+  echo WARNING ${CARGO_CONFIG} exists, moving away..
+  mv --backup=existing ${CARGO_CONFIG}{,.prev}
+}
+
+set -xueo pipefail
```

Comment:
Lets move this on line 2?

Reply:
I purposely moved this past the variable definitions and checks. Ideally we'd probably `set -ueo pipefail` at line 2 and only leave `set -x` here?

Reply:
Right, I'm mostly concerned about having `-ueo` part as early as possible